src/drivers/pwm_esc/pwm_esc.cpp

```diff
@@ -352,19 +352,26 @@ PWMESC::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS], unsigne
 	} else {
 		// In hitl, publish actuator_outputs_sim
 		// Only publish once we receive actuator_controls (important for lock-step to work correctly)
-		actuator_outputs_s actuator_outputs_sim{};
+
```

Comment:
extra linefeed (does this pass "make format?")

Reply:
i used `./Tools/astyle/fix_code_style.sh src/drivers/pwm_esc/pwm_esc.cpp` to fix format but somehow it missed that. will fix